src/main/java/com/ravlinko/concordion/extension/mockserver/executors/MockExecutor.java

```diff
@@ -6,44 +6,43 @@
 import org.mockserver.model.HttpRequest;
 import org.mockserver.model.HttpResponse;
 
-public class InitializationExecutor {
+public class MockExecutor {
 	private static final String MOCK_EXECUTOR_VARIABLE = "#mock";
 
 	private MockServerClient mockServerClient;
 
-	private InitializationExecutor(final String host, final int port) {
+	private MockExecutor(final String host, final int port) {
 		mockServerClient = new MockServerClient(host, port);
 	}
 
-	public static InitializationExecutor newExecutor(final Evaluator evaluator, final String host, final int port) {
-		InitializationExecutor initializer = new InitializationExecutor(host, port);
+	public static MockExecutor newExecutor(final Evaluator evaluator, final String host, final int port) {
+		MockExecutor initializer = new MockExecutor(host, port);
 		evaluator.setVariable(MOCK_EXECUTOR_VARIABLE, initializer);
 		return initializer;
 	}
 
-	public static InitializationExecutor fromEvaluator(Evaluator evaluator) {
-		return (InitializationExecutor) evaluator.getVariable(MOCK_EXECUTOR_VARIABLE);
+	public static MockExecutor fromEvaluator(Evaluator evaluator) {
+		return (MockExecutor) evaluator.getVariable(MOCK_EXECUTOR_VARIABLE);
 	}
 
 	private HttpRequest httpRequest;
 	private HttpResponse httpResponse;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)